tests/unit/worker/task/test_static_source_table.py

```diff
@@ -30,12 +30,10 @@ async def test_get_task_description(catalog):
             ),
         ),
     )
-    task = StaticSourceTableTask(
-        task_id=uuid4(),
-        payload=payload.dict(by_alias=True),
-        progress=Mock(),
-        app_container=Mock(),
-    )
+    app_container.override_instance_for_test("task_id", uuid4())
+    app_container.override_instance_for_test("progress", Mock())
+    app_container.override_instance_for_test("payload", payload.dict(by_alias=True))
+    task = app_container.get(StaticSourceTableTask)
```

Comment:
can retrieve task from app_container in test as well now instead of having to manually initialize